recipes/libalx-base/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+import os
+from conans import ConanFile, tools
+
+
+class libalx_base_conan(ConanFile):
```

Comment:
Weird casing for python 

```suggestion
class LibalxConanFile(ConanFile):
```